app/models/user.py

```diff
@@ -489,22 +489,11 @@ def __init__(self, _dict):
 
     @classmethod
     def create(
-        cls,
-        invite_from_id,
-        service_id,
-        email_address,
-        permissions,
-        auth_type,
-        folder_permissions,
+        cls, invite_from_id, service_id, email_address, permissions, auth_type, folder_permissions, is_join_request
```

Comment:
This has so many arguments now that it would be good to enforce keywords arguments (this avoid accidentally mixing them up)
```suggestion
        cls, *, invite_from_id, service_id, email_address, permissions, auth_type, folder_permissions, is_join_request
```